src/main/resources/application-dev.yml

```
@@ -62,4 +62,7 @@ logging:
     org.springframework.security: DEBUG
     org.springframework.security.oauth2: DEBUG
     org.springframework.web.client.RestTemplate: DEBUG
-    com.example.dnd_13th_9_be: DEBUG
\ No newline at end of file
+    com.example.dnd_13th_9_be: DEBUG
+
+aes:
+  private-key: ${AES_PRIVATE_KEY}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)